site/zh/blog/new-in-devtools-106/index.md

```diff
@@ -22,32 +22,46 @@ draft: true
 {% YouTube id='5gBqTXctxO8' %}
 
 <!-- start: translation instructions -->
+开始：翻译使用说明
 <!-- + 1. Remove the "draft: true" tag above when submitting PR -->
+1.当提交 PR 时去掉上面的“draft: true”标签
 <!-- + 2. Provide translations under each of the English commented original content -->
+2.提供每条英文评论原文下的翻译
 <!-- + 3. Translate the "description" tag above -->
+3.翻译上面的“描述”标签
 <!-- + 4. Translate all the <img> alt text -->
+4.翻译所有 <img> alt 文本
 <!-- + 5. Update the whats-new.md file -->
+5.更新 whats-new.md 文件
 
 <!-- ## Group files by Authored / Deployed in the Sources panel {: #authored } -->
+在 Sources 面板中按照 Authored / Deployed 对文件进行分组 {: #authored }
```

Comment:
```suggestion
## 在 Sources 面板中按照 Authored / Deployed 对文件进行分组 {: #authored }
```

Please add the heading # as part of the translation. The page will style these as section title.